rootfs/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM alpine:3.1
+FROM alpine:3.3
+
+# install git and configure gituser
+ENV GITHOME /home/git
+ENV GITUSER git
+# this is so the minio client (https://github.com/minio/mc) works properly
+ENV DOCKERIMAGE=1
+ENV DEIS_RELEASE 2.0.0-dev
```

Comment:
> What if no changed occurred that release

Maybe not in the `go` code but if a new version of openssl is available (to fix one vulnerability) the image will not reflect this change because of the cached layers.

That said, please tell me what to do with the order of the variables